src/assets/src/hooks/useGoogleAnalytics.ts

```diff
@@ -1,22 +1,72 @@
 import { useState, useEffect } from 'react';
 import GoogleAnalytics from 'react-ga4';
 import { useLocation } from 'react-router-dom';
+import { useOneTrust } from './useOneTrust';
+
 
 export const useGoogleAnalytics = (googleAnalyticsId?: string, debug?: boolean) => {
     let location = useLocation();
+
     const [initialized, setInitialized] = useState(false);
     const [previousPage, setPreviousPage] = useState(null as string | null);
 
     if (googleAnalyticsId && !initialized) {
+        GoogleAnalytics.gtag("consent", "default", {
+            ad_storage: "denied",
+            analytics_storage: "denied",
+            functionality_storage: "denied",
+            personalization_storage: "denied",
+            ad_user_data: "denied",
+            ad_personalization: "denied",
+            wait_for_update: 500
+        });
         setInitialized(true);
         GoogleAnalytics.initialize(googleAnalyticsId, { testMode: debug });
     }
 
+    const [loaded, oneTrustActiveGroups, setOptanonWrapper] = useOneTrust();
+    if (loaded) {
+        console.log("loaded oneTrustActiveGroups" + oneTrustActiveGroups);
```

Comment:
I'd probably lower this to` console.debug` if you want to have some type of indication in the frontend still.

Reply:
removed the console logs for now